.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.enabled": false
```

Comment:
.vscode folder should be ignored (add to .gitignore)

Reply:
Yep, but I also figured you wouldn't merge this PR anyway due to the review comments, so I didn't worry about it. 😄